inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -1323,29 +1322,47 @@ MKLDNNNode* MKLDNNNode::NodesFactory::create(const std::shared_ptr<ngraph::Node>
     return newNode;
 }
 
-bool MKLDNNNode::canBePerformedAsScaleShift() const {
-    bool inputsIsConst = true;
-    for (size_t i = 1; i < getParentEdges().size(); i++) {
-        if (!getParentEdgeAt(i)->getParent()->isConstant() || getParentEdgeAt(i)->getParent()->getType() != Input) {
-            inputsIsConst = false;
+bool MKLDNNNode::canBePerformedAsScaleShift(const MKLDNNNode *parentNode) const {
+    size_t portIntoFuse = 0;
+    for (size_t i = (parentNode == nullptr ? 1 : 0); i < getParentEdges().size(); i++) {
+        MKLDNNNode *node = getParentEdgeAt(i)->getParent().get();
+        if (node == nullptr) {
+            THROW_IE_EXCEPTION << "Cannot get parent node for " << getName() << " on " << i << " port";
+        }
+        if (node == parentNode) {
+            portIntoFuse = i;
+            continue;
+        }
+        if (!node->isConstant() || node->getType() != Input) {
```

Comment:
Why should we check "node->getType() != Input"?

Reply:
when we call ```fillScalesAndShifts``` we use method ```getConstBlob``` to fill internal buffers